tests/scene/test_text_edit.h

```diff
@@ -7717,7 +7717,7 @@ TEST_CASE("[SceneTree][TextEdit] viewport") {
 	CHECK(text_edit->get_h_scroll() == 0);
 
 	text_edit->set_h_scroll(10000000);
-	CHECK(text_edit->get_h_scroll() == 306);
+	CHECK(text_edit->get_h_scroll() == 314);
```

Comment:
Maybe this can use some style values instead of being hardcoded?

Reply:
Yes, this is actually what i was thinking about, but i didn't check how it's calculating the base width and i never touched any of the unit test files before.